### PR TITLE
Idp4

### PR DESCRIPTION
**Pull request by jalauros**
Update project from shib 3 to shib4. Ready to be shipped for further testing.

### CODE REVIEW

File: idp-stepup-api/pom.xml [hunk 1]
```diff
@@ -1,48 +1,62 @@
...
-                <artifactId>maven-compiler-plugin</artifactId>
-                <configuration>
-                    <source>${maven.compiler.source}</source>
-                    <target>${maven.compiler.target}</target>
-                </configuration>
-            </plugin>
-        </plugins>
-    </build>
+<?xml version="1.0"?>
+<!-- The MIT License Copyright (c) 2015, 2019 CSC - IT Center for Science,
```
Comment (blankdots) [inline idp-stepup-api/pom.xml]:
```suggestion
<!-- The MIT License Copyright (c) 2015 - 2020 CSC - IT Center for Science, 
```

**Review by blankdots — COMMENTED** [review]
first round of review turned out a small miss, and it looks like most of the changes are updates to java 11 and shib4 (as advertised)
Need to put this in intellij to have a better idea of it, will come back after I looked at it in the editor.

File: idp-stepup-api/pom.xml (hunk 1 above)
Comment (jalauros) [reply idp-stepup-api/pom.xml]: There are certainly also other license year misses. I can fix them all based on this comment.

**Comment by blankdots** [comment]
Nothing else. LGTM. Approved (if i would have access to the approve button)

**Pull request merged**